test/linter/test-versions.ts

```diff
@@ -90,7 +67,12 @@ const isValidVersion = (
   version: VersionValue,
 ): boolean => {
   if (typeof version === 'string') {
-    return validBrowserVersions[browser].includes(version);
+    if (version === 'preview') {
+      return !!browsers[browser].preview_name;
+    }
+    return Object.keys(browsers[browser].releases).includes(
+      version.replace('≤', ''),
+    );
```

Comment:
I think this can be simplified/optimized:

```suggestion
    return Object.hasOwn(browsers[browser].releases, version.replace('≤', ''));
```